src/provider/index.ts

```diff
@@ -162,14 +163,7 @@ export class Main {
             } else {
                 // There are no appWindows in the model with the same app uuid - Produce minimal appInfo from window information
                 const application = fin.Application.wrapSync(identity);
```

Comment:
Consider moving this line inside `getApplicationInfo`? Just so our interaction with the low-level OpenFin API is self contained